ui/app/AppLayouts/Wallet/addaccount/panels/WatchOnlyAddressSection.qml

```diff
@@ -26,7 +26,7 @@ Column {
         maximumHeight: Constants.addAccountPopup.itemHeight
         minimumHeight: Constants.addAccountPopup.itemHeight
         label: qsTr("Ethereum address or ENS name")
-        placeholderText: "0x95222293DD7278Aa3Cdd389Cc1D1d165CCBAfe5"
+        placeholderText: "Type or paste ETH address"
```

Comment:
```suggestion
        placeholderText: qsTr("Type or paste ETH address")
```

Reply:
it is not the first one! Thanks for review, fixed